### ISSUE TITLE
Add new exercise: markdown

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/javascript#918]
[problem specification ](https://github.com/exercism/problem-specifications/tree/master/exercises/markdown)| [description](https://github.com/exercism/problem-specifications/blob/master/exercises/markdown/description.md) | [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/markdown/canonical-data.json)

You may find it helpful to follow [this checklist](https://github.com/exercism/javascript/blob/83feda2af955df1ef9f3a8080338f506f0fdf696/CONTRIBUTING.md#new-exercise) from the [CONTRIBUTING.md](https://github.com/exercism/javascript/blob/83feda2af955df1ef9f3a8080338f506f0fdf696/CONTRIBUTING.md) resource.

**Comment by Kmelow** [comment]
Hello,

I have a question.

After successfully populating the `.meta/tests.toml` with `./bin/configlet sync --update`, is there a way to make the same for the tests in `<slug>.spec.js`?

Thanks,

**Comment by junedev** [comment]
@Kmelow No, the JavaScript track does not have a test case generator. You need to write the test file yourself.

**Comment by SleeplessByte** [comment]
This was done.
